libs/libcommon/src/libcommon/constants.py

```diff
@@ -43,6 +43,8 @@
 LARGE_MAX_FAILED_RUNS = 30  # for errors that should not be permanent
 MAX_FAILED_RUNS_PER_ERROR_CODE = {
     # default
+    "ConfigNamesError": DEFAULT_MAX_FAILED_RUNS,  # <- 20240822: to recompute all of them in the next backfill
```

Comment:
so you should remove this after the next backfill ?

Reply:
yes. Anyway, it can wait for some days, it will retry 3 times at most, and we don't have that many cases.